lunabot_control/scripts/excavation_controller.py

```diff
@@ -115,14 +115,19 @@ def publish_effort(self):
         effort_msg.lead_screw = self.constrain(
             self.lead_screw_voltage, max_percent=self.max_lead_screw_percent
         )
-        effort_msg.excavate = self.constrain(
+        effort_msg.excavate = self.constrain_RPM(
             self.exc_voltage, max_percent=self.max_exc_percent
         )
         self._effort_pub.publish(effort_msg)
 
     def constrain(self, val, max_percent):
         val = np.clip(-1, val, 1)  # Clipping speed to not go over 100%
-        return np.int8(val * 127 * max_percent)
+        return np.int32(val * 127 * max_percent)
+    
+    def constrain_RPM(self, val, max_percent):
+        val = np.clip(-1, val, 1)  # Clipping speed to not go over 100%
+        return np.int32(val * 1000 * max_percent)
```

Comment:
Why is RPM constrained to never go over 1000? It should be either 3000 or 5000. The percentage that we're spinning the motor at should be the thing that changes. 
127 was 100% of motor power last year.

Reply:
Updated in fix threshold commit